clinica/iotools/utils/data_handling.py

```diff
@@ -639,3 +655,298 @@ def center_all_nifti(bids_dir, output_dir):
+        center_width = max(len('Coordinate of center') + 3,
+                           3 + max(len(str(center)) for center in centers))
+
+        warning_message = Fore.YELLOW + '[Warning] It appears that ' + str(len(list_non_centered_files)) + ' files ' \
+                          + 'have a center way out of the origin of the world coordinate system. SPM has a high prob' \
+                          + 'ability to fail on these files:\n\n'
+        warning_message += ('%-' + str(file_width) + 's%-' + str(center_width) + 's%-s') % ('File',
+                                                                                            'Coordinate of center',
+                                                                                            'Distance to origin')
+        # 18 is the length of the string 'Distance to origin'
+        warning_message += '\n' + '-' * (file_width + center_width + 18) + '\n'
+        for file, center, l2 in zip(list_non_centered_files, centers, l2_norm):
+            # Nice formatting as array
+            # % escape character
+            # - aligned to the left, with the size of the column
+            # s = string, f = float
+            # . for precision with float
+            # https://docs.python.org/2/library/stdtypes.html#string-formatting for more information
+            warning_message += ('%-' + str(file_width) + 's%-' + str(center_width) + 's%-25.2f\n') % (basename(file),
+                                                                                                      str(center),
+                                                                                                      l2)
+
+        warning_message += '\nClinica provides a tool to counter this problem by replacing the center of the volume' \
+                           + ' at the origin of the world coordinates.\nUse the following command line to correct the '\
+                           + 'header of the faulty NIFTI volumes in a new folder:\n' + Fore.RESET \
+                           + Fore.BLUE + '\nclinica iotools center-nifti ' + abspath(bids_dir) + ' ' \
+                           + abspath(bids_dir) + '_centered\n\n' + Fore.YELLOW \
+                           + 'You will find more information on the command by typing ' + Fore.BLUE \
+                           + 'clinica iotools center-nifti' + Fore.YELLOW + ' in the console.\nDo you still want to '\
+                           + 'launch the pipeline now ?' + Fore.RESET
+        cprint(warning_message)
+        while True:
+            cprint('Your answer [yes/no]:')
+            answer = input()
+            if answer.lower() in ['yes', 'no']:
+                break
+            else:
+                cprint(Fore.RED + 'You must answer yes or no' + Fore.RESET)
+        if answer.lower() == 'no':
+            cprint(Fore.RED + 'Clinica will now exit...' + Fore.RESET)
+            sys.exit(0)
```

Comment:
Is a timeout needed needed?

Reply:
I think it is really important that the user see this message. If we launch the pipeline anyway, the user may never read this message and struggle to understand why his segmentations have failed.
I'd rather leave it that way

Reply:
Ok.